vlo-importer/src/test/java/eu/clarin/cmdi/vlo/importer/MetadataImporterTest.java

```diff
@@ -55,13 +55,13 @@ public void testImporterSimple() throws Exception {
         List<SolrInputDocument> docs = importData(sessionFile);
         assertEquals(1, docs.size());
         SolrInputDocument doc = docs.get(0);
-        assertEquals("testID1Session", getValue(doc, FacetConstants.FIELD_ID));
-        assertEquals("CollectionName", getValue(doc, FacetConstants.FIELD_COLLECTION));
-        assertEquals("testRoot", getValue(doc, FacetConstants.FIELD_DATA_PROVIDER));
-        assertEquals("kleve-route", getValue(doc, FacetConstants.FIELD_NAME));
-        assertEquals(sessionFile.getAbsolutePath(), getValue(doc, FacetConstants.FIELD_FILENAME));
-        assertEquals("video/x-mpeg1", getValue(doc, FacetConstants.FIELD_FORMAT));
-        assertEquals("video/x-mpeg1|../Media/elan-example1.mpg", getValue(doc, FacetConstants.FIELD_RESOURCE));
+        assertEquals("testID1Session", getValue(doc, fieldNameService.getFieldName(KEY.FIELD_ID)));
```

Comment:
For readability it might make sense to add a convenience method `getValue(SolrDocument, KEY)` in `ImporterTestCase` and use that instead of calling the `fieldNameService` directly over and over again in this and other test classes.